randovania/layout/preset_migration.py

```diff
@@ -6,7 +6,7 @@
 from randovania.layout.base.dock_rando_configuration import DockRandoMode, DockTypeState
 from randovania.lib import migration_lib
 
-CURRENT_VERSION = 33
+CURRENT_VERSION = 34
```

Comment:
you'll need to adjust the new migration to prevent merge conflicts, since I bumped the preset version

Reply:
Merged upstream into my main while using the github tool to fix merge conflicts in preset describer, preset migration, and dread configuration. Also fixed an oopsie in function ordering that I made. I would recommend testing a game export before accepting the PR however, as I am unable to do so on my current computer setup.